From 42916a9fae734c3a0b3dcc8d90f77669378a20c2 Mon Sep 17 00:00:00 2001
From: Neon <yyk1249501542@gmail.com>
Date: Tue, 12 Nov 2024 17:55:27 +0800
Subject: [PATCH 13/14] fix features wrongly opened

---
 crates/mega_panel/Cargo.toml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/crates/mega_panel/Cargo.toml b/crates/mega_panel/Cargo.toml
index af520b2100..d959d761a4 100644
--- a/crates/mega_panel/Cargo.toml
+++ b/crates/mega_panel/Cargo.toml
@@ -11,8 +11,8 @@ path = 'src/mega_panel.rs'
 workspace = true
 
 [dependencies]
-gpui = { workspace = true, features = ['test-support']}
-workspace = { workspace = true, features = ["test-support"]}
+gpui.workspace = true
+workspace.workspace = true
 
 mega.workspace = true
 worktree.workspace = true
@@ -26,3 +26,7 @@ util.workspace = true
 anyhow.workspace = true
 serde.workspace = true
 schemars.workspace = true
+
+[dev-dependencies]
+gpui = { workspace = true, features = ['test-support']}
+workspace = { workspace = true, features = ["test-support"]}
-- 
2.43.0

